Explorer/Assets/DCL/Multiplayer/SDK/Systems/GlobalWorld/PlayerCRDTEntitiesHandlerSystem.cs

```diff
@@ -48,7 +45,7 @@ protected override void Update(float t)
 
         [Query]
         [All(typeof(Profile))]
-        [None(typeof(PlayerCRDTEntity), typeof(DeleteEntityIntention))]
+        [None(typeof(PlayerCRDTEntity), typeof(DeleteEntityIntention), typeof(PlayerComponent))]
         private void AddPlayerCRDTEntity(in Entity entity, ref CharacterTransform characterTransform)
```

Comment:
Perhaps we should rename it into AddPlayerCRDTEntityForPeers

Reply:
Renamed to `AddRemotePlayerCRDTEntity`